torch/linalg/__init__.py

```diff
@@ -367,6 +367,91 @@
             [-3.1113,  2.7381]], dtype=torch.float64)
 """)
 
+lstsq = _add_docstr(_linalg.linalg_lstsq, r"""
```

Comment:
This op needs to be added to linalg.rst to appear in the docs:

https://github.com/pytorch/pytorch/blob/master/docs/source/linalg.rst

Reply:
Other than updating the .rst the docs overall look good and shouldn't block this PR. All the suggestions can be implemented in a follow-up.